lib/base.rb

```diff
@@ -14,7 +14,6 @@ def make_request_url url, options
     end
 
     def builder_options options
-      options.merge!(:api_user => @api_user, :api_key => @api_key)
```

Comment:
Is api_user not used anymore?

Reply:
Should probably just remove it if it isn't necessary https://github.com/OneHQ/sendgrid_webapi/blob/bc07fb40a43e1ad901a28a11fe92a5e3fca9a5e1/lib/base.rb#L7-L8